pkg/controllers/securitypolicy/securitypolicy_controller_test.go

```diff
@@ -263,11 +262,7 @@ func TestSecurityPolicyReconciler_GarbageCollector(t *testing.T) {
 		a.Items[0].UID = "1234"
 		return nil
 	})
-	go func() {
-		time.Sleep(1 * time.Second)
-		cancel <- true
-	}()
-	r.GarbageCollector(cancel, time.Second)
+	r.CollectGarbage(ctx)
```

Comment:
In UT, do we need to do some mock for common.GcOnce(r) when calling Reconcile.

```
func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
	// Use once.Do to ensure gc is called only once
	common.GcOnce(r)
```
Since GcOnce will also start go a goroutine.

Reply:
ok